demo/EuroBioC2023/workflow.Rmd

```diff
@@ -0,0 +1,128 @@
+
+![Figure source: Moreno-Indias et al. (2021) Statistical and Machine Learning Techniques in Human Microbiome Studies: Contemporary Challenges and Solutions. Frontiers in Microbiology 12:11.](figures/microbiome.png){width="400"}
+
+## Motivation
+
+### Microbiome research
+
+- Microbiome is a composition of microbes in well-defined area (gut, skin, mouth...)
+- Bilateral interaction between human and microbiome --> affects both health and disease.
+- The research is based on sequencing (characterization of genes and species).
+- Nowadays, multiomics approach is more common (integration of taxonomy information with metabolite data, for example)
+- Computational methods are the new microscope
+- The research has expanded rapidly in previous years
+
+```{r pubmed_fig, fig.width=6, fig.cap="PubMed publications per year with a search term 'microbiome' (fetched: Sep 5, 2023)", echo=FALSE}
+# Plot publication graph
+path <- "data/PubMed_Timeline_Results_by_Year.csv"
+df <- read.csv(path, skip = 1)
+
+library(ggplot2)
+x <- "Year"
+y <- "Count"
+
+plot <- ggplot(df, aes(x = .data[[x]], y = .data[[y]])) +
+    geom_bar(stat="identity")
+plot
+```
+
+### Big data
+
+- Add here something about cohort studies, big data, how analysis is done with big data, requirements...
+
+## miaverse (MIcrobiome Analysis) ![](figures/mia_logo.png){width="50"}
+
+- A framework for microbiome analytics
+    - [mia (analysis)](https://bioconductor.org/packages/release/bioc/html/mia.html)
+    - [miaViz (visualization)](https://bioconductor.org/packages/release/bioc/html/miaViz.html)
+    - [miaSim (simulation)](https://bioconductor.org/packages/release/bioc/html/miaSim.html)
+    - [Orchestrating Microbiome Analysis, OMA (tutorial book)](https://microbiome.github.io/OMA/)
+- Based on _TreeSummarizedExperiment (TreeSE)_ class
+    - Supports hierarchical data
+    - _phyloseq_ class is a subset of _TreeSE_
+    - Extension of _SingleCellExperiment_ class
+    - miaverse is compatible with other _SummarizedExperiment_ frameworks
+- Purpose to offer tools and tips for microbiome data analysis
+- Enables development of versatile analytical workflows in microbiome data science
+    - Supports multiomics (_MultiAssayExperiment_ class)
+    - Scalable
+    - Standardized
+
+![The structure of the TreeSummarizedExperiment (TreeSE) class.](figures/treese.png){width="700"}
+
+## The workflow
+
+### Importing the dataset
+
+We get the data from [MGnify database](https://www.ebi.ac.uk/metagenomics). It
+is a EMBL-EBI's database for metagenomic data. This large microbiome database
+can be accessed with _MGnifyR_ package which nowadays support _TreeSE_. The
+package will be submitted to Bioconductor's next release.
+
+We chose this dataset...
+
+As loading takes some time, the dataset is already loaded.
+
+For other available datasets and importing methods, check [OMA](https://microbiome.github.io/OMA/containers.html#example-data).
+
```

Comment:
Create a workflow e..g. with mia dataset. --> Let's switch the dataset later.

1. Summarize the data
2. Subset the data
3. Create plots with miaViz
4. Perform dbRDA
5. Some basic DAA

Reply:
How about demo on splitByRanks?